src/bundle/Resources/config/services/components.yml

```diff
@@ -0,0 +1,10 @@
+services:
+    ezplatform.matrix.fieldtype.javascripts:
+        parent: EzSystems\EzPlatformAdminUi\Component\TwigComponent
+        autowire: true
+        autoconfigure: false
+        public: false
+        arguments:
+            $template: '@@ezdesign/matrix_fieldtype/javascripts.component.html.twig'
```

Comment:
`javascripts_component.html.twig`? We use underscore in filenames.

Reply:
This is the name we use in few places already. Blame frontend!